src/unit_tests/wazuh_db/test_wdb_global.c

```diff
@@ -4086,9 +4086,27 @@ void test_wdb_global_select_groups_exec_fail(void **state)
     will_return(__wrap_wdb_begin2, 1);
     will_return(__wrap_wdb_stmt_cache, 1);
 
-    will_return(__wrap_wdb_exec_stmt, NULL);
+    /* wdb_exec_stmt_sized */
+    wrap_wdb_exec_stmt_sized_failed_call(STMT_MULTI_COLUMN);
     will_return(__wrap_sqlite3_errmsg, "ERROR MESSAGE");
-    expect_string(__wrap__mdebug1, formatted_msg, "wdb_exec_stmt(): ERROR MESSAGE");
+    expect_string(__wrap__mdebug1, formatted_msg, "Failed to get groups: ERROR MESSAGE.");
+
+    result = wdb_global_select_groups(data->wdb);
+
+    assert_null(result);
+}
+
+void test_wdb_global_select_groups_socket_full(void **state)
+{
+    cJSON *result = NULL;
+    test_struct_t *data  = (test_struct_t *)*state;
+
+    will_return(__wrap_wdb_begin2, 1);
```

Comment:
Since wdb_begin2 is evaluate as `(wdb_begin2(wdb) < 0) { return NULL} ` Wouldn't be more readable to change 1 by OS_SUCCESS?.

Reply:
It'd be more correct, yes !
Fixed in some tests